app/main/posts/detail/detail.html

```diff
@@ -48,9 +48,9 @@ <h1 class="form-sheet-title survey-title">
           ng-if="isPostValue(key) && form_attributes[key] && showType(form_attributes[key].type)"
           class="postcard-field"
           >
-
           <post-media-value
             ng-if="form_attributes[key].type === 'media'"
+            media-has-caption="form_attributes[key].config.hasCaption"
```

Comment:
Does this fail weirdly if `config.hasCaption` doesn't exist?

Reply:
nope. The only reason it could throw an error would be if config was NULL for the media field, which isn't the default behavior for the config field so it really isn't worth cluttering the code with a ternary in there. Also, IF at some point we had messed up and the config could be NULL in some databases, our migration solves that.